hooks/queries/product/useGetProducts.ts

```diff
@@ -0,0 +1,29 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import { productAPI } from 'apis';
+
+const LIMIT = 10;
+
+const getProductsAPI = async ({ pageParam = 0 }) => {
+  const { data } = await productAPI.getProducts({ offset: pageParam });
+
+  return {
+    data,
+    current_page: pageParam,
+    isLast: data.length === 0 ? true : false,
+  };
+};
+
+const useGetProducts = () => {
+  return useInfiniteQuery(['products'], getProductsAPI, {
+    getNextPageParam: (lastPage) => {
+      if (!lastPage.isLast) {
+        return lastPage.current_page + LIMIT;
+      }
+
+      return undefined;
```

Comment:
저는 if문 안에 조건문이 `!`연산자가 있는 것보다는 없이 바로 읽히는게 좋을 것 같다고 생각이 들었습니다! 다른 분들은 어떻게 생각하시나요?

```suggestion
      if (lastPage.isLast) {
        return undefined;
      }
      return lastPage.current_page + LIMIT;
```

Reply:
좋은 것 같습니다.